charts/rancher-istio/100.0.0+up1.10.4/Chart.yaml

```diff
@@ -2,6 +2,7 @@ annotations:
   catalog.cattle.io/auto-install: rancher-kiali-server-crd=100.0.0+up1.35.0
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: Istio
+  catalog.cattle.io/kube-version: '>= 1.18.0-0 < 1.22.0-0'
```

Comment:
one more thing; this currently indicates we should not be able to install it on a 1.22 or less cluster, but the issue seems to indicate we only need to drop support for 1.23+? https://github.com/rancher/rancher/issues/37697

Also, why are we limiting to 1.18+ / where does that requirement come from?